dash/dash.py

```diff
@@ -676,6 +679,10 @@ def _generate_meta_html(self):
 
     # Serve the JS bundles for each package
     def serve_component_suites(self, package_name, path_in_package_dist):
+        (path_in_package_dist, has_fingerprint) = check_fingerprint(
```

Comment:
non-🚫 - you don't need the parens
```suggestion
        path_in_package_dist, has_fingerprint = check_fingerprint(
```